src/Menu/StyledMenuItem.js

```diff
@@ -13,6 +13,7 @@ const darker = (theme, base) => {
 const baseStyles = ({ theme, baseColor }) => css`
   color: ${theme.colors.text.default};
   padding: 0 ${theme.space.regular};
+  white-space: nowrap;
```

Comment:
damian suggested this. styled-system does not have an export for the white-space directive

Reply:
that's specifically verboten in the story